.github/workflows/build.yml

```diff
@@ -1,7 +1,7 @@
 name: "Indy-VDR"
 
 env:
-  RUST_VERSION: "1.81.0"
+  RUST_VERSION: "1.85.0"
```

Comment:
1.82 is the oldest supported, 1.90 is the latest, this is a happy medium with no breaking changes